src/Sarif/ExtensionMethods.cs

```diff
@@ -153,15 +153,19 @@ public static void Populate(this Region region, NewLineIndex newLineIndex)
             }
         }
 
+        public static string GetMessageText(this Result result, IRule rule)
+        {
+            return GetMessageText(result, rule, concise: false);
+        }
+
         public static string GetMessageText(this Result result, IRule rule, bool concise = false)
```

Comment:
You added the overload which is good, but now you have to remove the default parameter:

```
public static string GetMessageText(this Result result, IRule rule, bool concise)
{
    ...
```